modules/claims_api/lib/claims_api/v2/lighthouse_military_address_validator.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module V2
+    module LighthouseMilitaryAddressValidator
+      MILITARY_CITY_CODES = %w[
+        APO
+        FPO
+        DPO
+      ].freeze
+
+      MILITARY_STATE_CODES = %w[
+        AE
+        AP
+      ].freeze
```

Comment:
Do we also need `AA` — Armed Forces America ([source](https://www.easypost.com/us-military-shipping-addresses-apo-fpo-and-dpo))?

Reply:
@tycol7 got this and the other comment addressed with [e16fb16](https://github.com/department-of-veterans-affairs/vets-api/pull/18376/commits/e16fb163025b5da94cb71748f1ca3b406274e3c5)